bindings/matrix-sdk-crypto-ffi/src/error.rs

```diff
@@ -93,3 +93,25 @@ impl From<InnerStoreError> for DecryptionError {
         Self::Store { error: err.to_string() }
     }
 }
+
+#[cfg(test)]
+mod tests {
+
+    use super::*;
+
+    #[test]
+    fn test_withheld_error_mapping() {
+        use matrix_sdk_crypto::types::events::room_key_withheld::WithheldCode;
+
+        let inner_error = MegolmError::MissingRoomKey(Some(WithheldCode::Unverified));
+
+        let binding_error: DecryptionError = inner_error.into();
+
+        match binding_error {
```

Comment:
You can use the `assert_matches!()` macro here to get a better error message in case the test fails.

Reply:
Oh, I just discovered that it was possible to "capture" the match while doing assert_matches.
updated